site/layouts/shortcodes/note.html

```diff
@@ -1,6 +1,3 @@
 <aside class="note">
-    <div class="note-icon">
-        {{partial "svg/exclamation.svg" (dict "size" "20px" ) }}
-    </div>
-    <div class="note-content">{{- .Inner -}}</div>
+    <div class="note-content">{{ .Inner | markdownify }}</div>
```

Comment:
This is needed because of breaking changes in https://gohugo.io/news/0.55.0-relnotes/